common_test.go

```diff
@@ -56,7 +56,7 @@ func addSslOptions(cluster *ClusterConfig) *ClusterConfig {
 	return cluster
 }
 
-var initOnce sync.Once
+var initKeyspaceOnce = map[string]*sync.Once{}
```

Comment:
It is going to have parallel access, so you need to have a mutex to guard it.
Please make an struct with method that pulls sync.Once from it